models/labels.go

```diff
@@ -24,6 +24,7 @@ const (
 	// StateReasonAnnotation is the name of the annotation that explains the difference between evaluation state and alert state (i.e. changing state when NoData or Error).
 	StateReasonAnnotation = GrafanaReservedLabelPrefix + "state_reason"
 
-	ValuesAnnotation      = "__values__"
-	ValueStringAnnotation = "__value_string__"
+	ValuesAnnotation          = "__values__"
+	ValueStringAnnotation     = "__value_string__"
+	LogzioAccountIdAnnotation = "__logzioAccountId__" //LOGZ.IO GRAFANA CHANGE :: DEV-43657: set correct app urls on notifications
```

Comment:
does that mean we have a duplication of these in both repos? 🤔 cause i also needed to add this.
i wonder if we actually need to reference the same ticket in these. 😅

Reply:
yeah for this one i agree. i wonder mostly if in the data-viz-next we would want to maybe change this so it's easy to connect and see the 2 🤔